src/components/StatusUpdater/index.tsx

```diff
@@ -114,7 +115,28 @@ const StatusUpdaterProtected = ({ activity, current, fullWidth, user, thisOrg }:
     setConfirming(true);
   }
 
-  const actions = getStatusOptions(current, activity.startTime, activity.forceStandbyOnly, activity.earlySignInWindow);
+  const lastOrgId = participant?.timeline[0].organizationId;
+  const currentOrgId = thisOrg.id;
+  let actions;
+  if (isActive(current) && currentOrgId !== lastOrgId) {
+    let transition;
+    switch (current) {
+      case ParticipantStatus.Remote:
+        transition = statusTransitions.inTown;
+        break;
+      case ParticipantStatus.Standby:
+        transition = statusTransitions.standBy;
+        break;
+      default:
+        transition = statusTransitions.signIn;
```

Comment:
Could easily be missing something, but in your PR description you say that in an active state the only thing that happens is the user is signed out of the old org. But this status transition indicates they're being moved to signed in, no?

Reply:
Yeah, the second half of the description explains some use cases where I decided to handle the sign in for the user, after all.

Reply:
Right, that's what I'm referring to. In that paragraph you say:

> Signing Out while in an "Active" status => Only Sign Out of the previous org. (this was an unhandled edge case prior to this change)

But the code is reading to me that the user will be placed into a signed in state, not signed out.

Reply:
Ah, yeah, at this point the user hasn't actually changed their status yet. This is just just determining which transition options are _available_. Specifically, which sign-in option should be available if you are currently active with Unit A, but logged into the portal of Unit B. The sign out option is always available and is appended on Line 135, just below.

The logic that actually handles the sign-in/out is in the activity reducers. Basically, if the new status is from a different org, it signs them out of the previous org -- then if the new status is an active status it continues with the regular update in the current org. Sign out aborts the change in the current org because we replace it with a sign out for the previous org.

From the examples above, its the last scenario -- I am signed into CRT, but I'm logged into ESAR's Portal.
- I have a "Switch from CRT" option, which would sign me out of CRT and back in (to "In Town" in this example) as ESAR.
- I also have a "Sign Out from CRT" option that will just sign me out from the other org.

![image](https://github.com/KingCountySAR/respond-next/assets/11284884/e9a16a79-6e69-48e0-b28e-8b921fcf1cdf)

In this scenario I chose the secondary option from within the ESAR portal, and I am now signed out of CRT, even though I took the action from the ESAR portal.
![image](https://github.com/KingCountySAR/respond-next/assets/11284884/0f419a1a-9a22-493e-b1d7-82652f1fed29)